app/src/main/res/values/strings.xml

```diff
@@ -29,6 +29,13 @@
     <item quantity="one">Receiving shared content. Processing the image might take some time depending on the size of the image and your device</item>
     <item quantity="other">Receiving shared content. Processing the images might take some time depending on the size of the images and your device</item>
   </plurals>
+
+  <!--accessibility UI description strings-->
+  <string name="icon">Icon</string>
+  <string name="bookmark_Button_Image">Bookmark Button</string>
+  <string name="nearby_Item_Image">Nearby Item</string>
```

Comment:
I am a bit surpised by the uppercases after the underscores, is it the convention we use throughout the project?

Reply:
sorry for this silly mistake. I have rectified it and few more files added with content description @nicolas-raoul